src/models/Drink.py

```diff
@@ -0,0 +1,22 @@
+from bson import ObjectId
+from pprint import pformat
+
+class Drink:
+
+  def __init__(self, user_id: ObjectId, ingredients = [ ]) -> None :
+    self.user_id = user_id
+    self.ingredients = ingredients
+
+    # set _id fields
+    self._id: ObjectId = None
+    self.review_ids = set()  #ObjectIds of reviews on this drink
+
+  def __repr__(self) -> str:
+    data = pformat(vars(self))[1:-1]
+    return f"Drink<\n {data}\n>"
```

Comment:
Can you add a space right after Drink here please? Just for consistency between the models when printing them.